pyscriptjs/src/utils.ts

```diff
@@ -1,46 +1,44 @@
-function addClasses(element: HTMLElement, classes: Array<string>) {
+export function addClasses(element: HTMLElement, classes: string[]) {
     for (const entry of classes) {
         element.classList.add(entry);
     }
 }
 
-function removeClasses(element: HTMLElement, classes: Array<string>) {
+export function removeClasses(element: HTMLElement, classes: string[]) {
     for (const entry of classes) {
         element.classList.remove(entry);
     }
 }
 
-function getLastPath(str: string): string {
-    return str.split('\\').pop().split('/').pop();
+export function getLastPath(str: string): string | undefined {
```

Comment:
Why a possibility of `undefined` here?
I ask since -- `str.split('\\').pop()` will always return a string, even if the original string doesn't have any `\\` in it i.e. `"hh".split('\\').pop() == "hh"`

Reply:
We will get to that. This PR focuses on types affecting getAttribute only.

Reply:
looking again...

Reply:
Oh. the ?. in the next line

```ts
return str.split('\\').pop()?.split('/').pop();
```

Reply:
Yeah, I mean, why do we need the `?`

Reply:
fixed